travel-visas.md

```diff
@@ -0,0 +1,66 @@
+# Prepare for visa application (optional) 🛂
+
+A quick way to check if you may need a visa is to enter `wiki visa requirements
+for [your country here] citizens` into a search engine. The first result is typically a
+wikipedia page with a table of all countries' visa policy that would apply to you.
+
+If you need a visa, don't worry! You can still apply for travel funds. You'll need
+to account for extra time to prepare your visa application, apply for the visa,
+and receive it before the date of travel.
+
+## How much time will you need? 📅
+
+We've provided here a checklist of items that a visa application could require.
+Thinking about how long it will take you to prepare these will give you an idea of how
+long it could take to prepare your visa application.
+
+This is not a complete list! This is only a list of items that are typically requested
+when visas are being processed.
+
+Typical visa applications may ask for any or all of the following:
+
+* An original passport. If you don't have one, you'll need to apply for one first.
+Check with your passport issuing authority for the processing time.
+* Passport-sized photos.
+* An original proof of residency for the country where you live,
+assuming you are not a citizen. For example, if you live in Canada this could be
+a Permanent Residence Card or a Temporary Residence Visa.
+* Copies of any of the above documents.
+* A copy of your air ticket.
+* A copy of your accommodation reservations.
+* Proof of travel insurance.
+* A fee for processing your application.
+* An invitation letter. If it's an event hosted by the Foundation, you can use
+[this form][visa-request-form] to request a letter. The Foundation
+can help with visa issues, so please use the request form as far ahead as possible.
+For Collaborator Summits, which are not hosted by the Foundation, it cannot provide
+an invitation letter.
+
+For the actual requirements, check the website of the consulate of the country to
+which you will be traveling.
+
+## Visa processing times ⏱
+
+Check the visa application processing time and plan accordingly.
+
+## Which visa do you need? 🛂
+
+In some cases, a visitor or tourism or tourist visa is what you need. In other cases, you may
+need a business visa. Check the visa policy on the consulate website for what you
```

Comment:
Note that speaking at conferences almost always entails a business visa, attending one may or may not.

It would be nice if there is a note about a visa does not guarantee that you can enter the country. You'll still have to go through the border control, and the visa must match your answers to the questions from the border guard.